examples/ws2812/main.go

```diff
@@ -12,11 +12,12 @@ import (
 	"tinygo.org/x/drivers/ws2812"
 )
 
-var leds [10]color.RGBA
+var (
+	noe  machine.Pin
```

Comment:
This looks like a typo.
Did you test this code?

Reply:
There was a typo. Also, the variable "noe" wasn't used at all. I've fixed it to make it more understandable. @aykevl